modules/cnf-checking-numa-aware-scheduler-logs.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_module-type: PROCEDURE
+[id="cnf-checking-numa-aware-scheduler-logs_{context}"]
+= Checking the NUMA-aware scheduler logs
+
+Troubleshoot problems with the NUMA-aware scheduler by looking at the logs. If required, you can increase the scheduler log level by modifying the `spec.logLevel` field of the `NUMAResourcesScheduler` resource. Acceptable values are `Normal`, `Debug`, and `Trace`, with `Trace` being the most verbose option.
+
+[NOTE]
+====
+To change the log level of the secondary scheduler, delete the running scheduler resource and re-deploy it with the changed log level. The scheduler is unavailable for scheduling new workloads during this downtime.
+====
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Delete the running `NUMAResourcesScheduler` resource:
++
+[source,terminal]
+----
+$ oc delete NUMAResourcesScheduler
```

Comment:
need to provide the resource name:
`[root@registry ~]# oc get NUMAResourcesScheduler 
NAME                     AGE
numaresourcesscheduler   90m
[root@registry ~]# oc delete NUMAResourcesScheduler numaresourcesscheduler
numaresourcesscheduler.nodetopology.openshift.io "numaresourcesscheduler" deleted
[root@registry ~]# 
`